shared/middleware/helpers/auth_helpers.rb

```diff
@@ -35,7 +35,11 @@ def get_user_sharing_disabled(user_id)
   user_properties = DASHBOARD_DB[:users].select(:properties).first(id: user_id)
   # If the user does not exist, return true.
   return true unless user_properties
-  parsed_properties = JSON.parse(user_properties[:properties])
+  get_sharing_disabled_from_properties(user_properties[:properties])
+end
+
+def get_sharing_disabled_from_properties(properties)
```

Comment:
in light of recent discussion on slack, we should really have a simple unit test on both of these auth helper methods too.